gluoncv/data/tracking_data/track.py

```diff
@@ -0,0 +1,779 @@
+""" siamrpn dataloader,include youtube-bb,VID,DET,COCO dataset
+Code adapted from https://github.com/STVIR/pysot """
+# coding: utf-8
+# pylint: disable=missing-docstring,unused-argument,arguments-differ
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import logging
+import os
+import numpy as np
+
+from mxnet.gluon.data import dataset
+from gluoncv.utils.filesystem import try_import_cv2
+from gluoncv.model_zoo.siamrpn.siamrpn_tracker import corner2center, center2corner
+from gluoncv.model_zoo.siamrpn.siamrpn_tracker import Corner, Center, Anchors
+
+def IoU(rect1, rect2):
```

Comment:
Is this IoU any different from other IoU functions? Try to adopt existing IoU functions. 

If this one is different, please put it in `gluoncv/utils/metrics/tracking.py` because that file should contain all the metric functions.

Reply:
The IoU between two anchor，gluoncv/utils/bbox/bbox_iou  between two numpy anchor used np Broadcasting，so  I will put it in  ` gluoncv/utils/metrics/tracking.py`